app/containers/message/Message.js

```diff
@@ -101,6 +107,14 @@ const MessageTouchable = React.memo((props) => {
 });
 MessageTouchable.displayName = 'MessageTouchable';
 
+const ReadReceipt = React.memo(({ Message_Read_Receipt_Enabled, unread }) => {
+	if (Message_Read_Receipt_Enabled) {
+		return !unread ? <CustomIcon name='check' color={COLOR_PRIMARY} size={15} /> : <View style={styles.emptySpace} />;
+	}
+	return null;
+});
+ReadReceipt.displayName = 'ReadReceipt';
```

Comment:
`displayName` makes React to show the name in React Devtools.
Here we are using `Message*` pattern as this is a huge component.
Rename it to `MessageReadReceipt`.